### DISCUSSION TITLE
ToString Help

### DISCUSSION THREAD

**Discussion by Billy4896** [Billy4896/Tafe230123#4] [General]
Hi all,

I have a technical question surrounding the use of toString("C). It only goes to 4 decimal places when I need it to vary on the conversion. Any suggestions?

Thanks,

Billy

**Comment by jamiebenson1** [comment]
Hey Billy, 
ive had a look on stack overflow and found this, if you scroll down on the link someone answers it fairly well.
https://stackoverflow.com/questions/164926/how-do-i-display-a-decimal-value-to-2-decimal-places
